admin/partials/wp-security-bp-admin-display.php

```diff
@@ -14,12 +14,115 @@
 ?>
 
 <!-- This file should primarily consist of HTML with a little bit of PHP. -->
-<h1>OLAKEASE</h1>
 <?php
 
-$users = get_users( array( 'include' => array( 2, 3 ) ) );
-foreach ( $users as $user ) {
-    if ( true ) {
-        var_dump( $user );
+class WP_Security_BP_Admin_Get_Users {
+
+	private $users;
+
+	public function __construct(){
+		$this->$users = get_users(); 
+	}
+	
+	public function check_danger_id_admin(){ 
+
+		$danger_ids = array();
```

Comment:
I would use the _not recommended ids_ (they really are not so dangerous 😄) as **an attribute** of the class.